src/services/continuedev/core/index.d.ts

```diff
@@ -618,17 +621,18 @@ export class SignatureInformation {
 	 * the UI.
 	 */
 	label: strin
-  /**
-   * The parameters of this signature.
-   */
-  parameters: ParameterInformation[];
-
-  /**
-   * The index of the active parameter.
-   *
-   * If provided, this is used in place of {@linkcode SignatureHelp.activeParameter}.
-   */
-  activeParameter?: number;g
+	/**
+	 * The parameters of this signature.
+	 */
+	parameters: ParameterInformation[]
+
+	/**
+	 * The index of the active parameter.
+	 *
+	 * If provided, this is used in place of {@linkcode SignatureHelp.activeParameter}.
+	 */
+	activeParameter?: number
+	g
```

Comment:
this seems accidental? This g should probably be on line 623? does this even compile?